FindPETSc.cmake

```diff
@@ -224,6 +235,10 @@ show :
   else ()
     set (PETSC_LIBRARY_VEC "NOTFOUND" CACHE INTERNAL "Cleared" FORCE) # There is no libpetscvec
     petsc_find_library (SINGLE petsc)
+    # Hack : for ubuntu with real in the architecture
+    if (PETSC_LIBRARY_SINGLE  STREQUAL "PETSC_LIBRARY_SINGLE-NOTFOUND")
```

Comment:
Better to use a boolean test instead of `STREQUAL`. But if they are distributing multiple versions like this, we need to also find a compatible `petscconf.h`. I don't see that, so your change here will produce broken code.